stdlib/ctypes/__init__.pyi

```diff
@@ -36,15 +36,25 @@ class CDLL(object):
     _name: str = ...
     _handle: int = ...
     _FuncPtr: Type[_FuncPointer] = ...
-    def __init__(
-        self,
-        name: Optional[str],
-        mode: int = ...,
-        handle: Optional[int] = ...,
-        use_errno: bool = ...,
-        use_last_error: bool = ...,
-        winmode: Optional[int] = ...,
-    ) -> None: ...
+    if sys.version_info < (3, 8):
```

Comment:
Could you put the 3.8+ branch first? That's the general convention and makes the code easier to follow.